aten/src/ATen/core/function_schema.cpp

```diff
@@ -41,9 +41,15 @@ FunctionSchema FunctionSchema::cloneWithRealTypes(bool with_symint) const {
     }
   };
   std::vector<Argument> new_arguments, new_returns;
-  std::transform(arguments().begin(), arguments().end(), std::back_inserter(new_arguments), cloneWithRealTypes);
+  new_arguments.reserve(arguments().size());
```

Comment:
Mind adding a comment here? Also, if size of the vector is know ahead of time, why not pre-allocate vector to given size on line 43 and just add elements at index?

Reply:
resize() will default-construct all the elements of the vector, which matters for a type like Argument with a non-trivial default constructor.

Reply:
> adding a comment

I don't think there is anything to be commented; we should *always* pre-allocate before building vectors if we know their size.